artifactory/utils/readremotefile.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/jfrog/jfrog-client-go/artifactory"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+)
+
+// Download and unmarshal a file from artifactory.
```

Comment:
There's probably no need have a dedicated file just for this util. Let's move this function into utils.go and remove tis file.